apps/omg_conformance/test/support/conformance/signatures_hashes.ex

```diff
@@ -132,8 +133,11 @@ defmodule Support.Conformance.SignaturesHashes do
 
   defp call_contract(contract, signature, args, return_types) do
     data = ABI.encode(signature, args)
+    from = Configuration.authority_address()
+
+    {:ok, return} =
+      Ethereumex.HttpClient.eth_call(%{from: from, to: Encoding.to_hex(contract), data: Encoding.to_hex(data)})
```

Comment:
you already have a contract address `to: Encoding.to_hex(contract)`. Could you not use the same address?

Reply:
```suggestion
      Ethereumex.HttpClient.eth_call(%{from: Encoding.to_hex(contract), to: Encoding.to_hex(contract), data: Encoding.to_hex(data)})
```
```suggestion
      Ethereumex.HttpClient.eth_call(%{from: from, to: Encoding.to_hex(contract), data: Encoding.to_hex(data)})
```